spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -278,18 +280,6 @@ independence.
 * They can access the large set of APIs
 available for the Java platform.
 
-=== Relationship to Jakarta EE Platform
```

Comment:
I think we should delete this section, as it is likely to be impossible to keep in sync. Servlet 5.0 will be part of the platform spec, but 5.1 is an unknown. It may end up in the next version of the platform, or we maybe have 5.2 by then. A spec may not be part of the platform when it is published, then included later.

Reply:
+1 The platform should say what and in which versions it includes things.